### ISSUE TITLE
npm deploys 3.3.2 css-files

### ISSUE THREAD

**Issue by drmonty** [thomaspark/bootswatch#431]
related to #429 3.3.4 npm-package includes css files from 3.3.2

Reproducable on linux with:

```
npm install bootswatch && head node_modules/bootswatch/flatly/bootstrap.css
```

will display:

```
bootswatch@3.3.4 node_modules/bootswatch
@import url("//fonts.googleapis.com/css?family=Lato:400,700,400italic");
/*!
 * bootswatch v3.3.2
 * Homepage: http://bootswatch.com
 * Copyright 2012-2015 Thomas Park
 * Licensed under MIT
 * Based on Bootstrap
*/
/*! normalize.css v3.0.2 | MIT License | git.io/normalize */
html {
```

**Comment by thomaspark** [comment]
This affects only Flatly, which failed to build 3.3.4 version due to missing semicolon. It's been fixed with `v3.3.4+1`.

Unfortunately, npm doesn't provide a way to republish a new build to 3.3.4, and does not recognize build numbers such as 3.3.4+1 (I'd love to be wrong about this). So best option is to manually update Flatly for time being, or use something like Bower which does recognize `3.3.4+1` as a newer release.

**Comment by drmonty** [comment]
Ah, alright :). Thought it is a general issue (not only flatly), but you're right.

Thanks :)!
